proto/streamlit/proto/MultiSelect.proto

```diff
@@ -29,4 +29,5 @@ message MultiSelect {
   bool set_value = 8;
   bool disabled = 9;
   LabelVisibilityMessage label_visibility = 10;
+  int32 max_selections = 11;
```

Comment:
Drive-by: I think there's a decent argument for not using the unsigned variant, because the underflow behavior is can lead to quite unexpected results. Like, if you subtract 1 from `uint 0`, you get `maxint`, I assume. -1 is an error value here, but so is `maxint` in most cases -- and it's non-obvious how you arrived at `maxint`, whereas a value of -1 - it's easy to understand how you got there.

I think using `uint` for bitmasks makes sense, where the expectation is that you'll never do arithmetic, but for normal "counting" values we should consider using signed int types even when the value should not be negative.

Reply:
(I'd feel differently if the arithmetic were checked, and raised an error when you try to store an unsigned int - but in this case, the `uint` type won't bring us any additional safety, it'll only lead to harder-to-debug failure cases. Even if protobuf does bounds checking on unsigned ints, TypeScript/JavaScript don't.)

Reply:
Ah, okay good point. I guess in that case should leave things as-is (along with a bunch of other things that are signed)